### PR TITLE
Requirebilling

### PR DESCRIPTION
**Pull request by ttnnkkrr**

### CODE REVIEW

File: js/securesubmit_plugin.js [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
     jQuery(function(){
         var scriptpath = jQuery("script[src]").last().attr("src").split('?')[0].split('/').slice(0, -5).join('/')+'/';
-        scriptpath += 'wp-content/plugins/securesubmit/template/index.html';
+        scriptpath += 'wp-content/plugins/donate/template/index.html';
```
Comment (slogsdon) [inline js/securesubmit_plugin.js]: This should be reverted at the very least to not break existing installations, but ideally, this should be updated to create this address on the PHP side via WordPress's API around discovering plugin URLs.

File: README.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ Screenshots
 
 Changelog
 ------------
+
+#### 1.5.3
+* Added support for requiring shipping and billing information
+* Test against PHP 7.0.10
```
Comment (slogsdon) [inline README.md]: Can we add a line above this stating that something to the effect of `Add support for PHP 7` to help clarify that we support PHP 7?

We should also make a note of the change in behavior where the fields themselves are now marked as required for user input.

File: SecureSubmit.php [hunk 3]
```diff
@@ -694,14 +703,16 @@ function shortcode($atts) {
                 // BILLING BODY
                 var <?php echo $prefix; ?>_billing_html = "<div id='<?php echo $prefix; ?>_billing_panel'>";
                 <?php echo $prefix; ?>_billing_html += "<div class='checkout-card-information'>Billing Information</div>";
-                <?php echo $prefix; ?>_billing_html += "<div class='card-number'><input type='text' name='cardholder_name' id='cardholder_name' class='checkout-input checkout-card' placeholder='Name on Credit Card'></div>";
-                <?php echo $prefix; ?>_billing_html += "<div class='card-number'><input type='text' name='cardholder_address' id='cardholder_address' class='checkout-input checkout-card' placeholder='Credit Card Billing Address'></div>";
+                <?php echo $prefix; ?>_billing_html += "<div class='card-number'><input type='text' name='cardholder_name' id='cardholder_name' class='checkout-input checkout-card <?php
+                    echo $billingRequired;
```
Comment (slogsdon) [inline SecureSubmit.php]: This echo should be all on one line to match the others.

File: SecureSubmit.php [hunk 4]
```diff
@@ -1682,7 +1778,7 @@ function submit_payment() {
 
         // modal
         if (isset($_POST['cardholder_name'])) {
-            list($first, $middle, $last) = split(' ', $_POST['cardholder_name']);
+            list($first, $middle, $last) = preg_split ("/ /", $_POST['cardholder_name']);
```
Comment (slogsdon) [inline SecureSubmit.php]: Why not use `explode` here to remove the need to search with a regex pattern?

File: js/securesubmit_plugin.js (hunk 1 above)
Comment (ttnnkkrr) [inline js/securesubmit_plugin.js]: @slogsdon Yea good catch i meant to change that

File: SecureSubmit.php (hunk 4 above)
Comment (ttnnkkrr) [inline SecureSubmit.php]: @slogsdon i guess you dont like regex?
i can change it to explode

Comment (slogsdon) [inline SecureSubmit.php]: I don't have anything against regex, but I don't see any benefit to using it here. I typically only use it if there's a need.

File: SecureSubmit.php (hunk 3 above)
Comment (SenthilKumar-86) [inline SecureSubmit.php]: Formatting fixed

File: README.md (hunk 2 above)
Comment (SenthilKumar-86) [inline README.md]: Readme file updated to address required fields

Comment (slogsdon) [inline README.md]: @SenthilKumar-86 Would you address the other part of my previous comment to clarify that the plugin supports PHP 7?

Comment (SenthilKumar-86) [inline README.md]: updated the read me file

**Pull request merged**